src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -121,6 +121,23 @@ def get(self, request: Request, group) -> Response:
 
         results = get_similar_issues_embeddings(similar_issues_params)
 
+        analytics.record(
+            "group_similar_issues_embeddings.count",
+            org_id=group.organization.id,
+            project_id=group.project.id,
+            group_id=group.id,
+            count_over_threshold=len(
+                [
+                    result["stacktrace_similarity"]  # type: ignore
```

Comment:
we can ignore the typing error for these because `result` can be of type `SimilarIssuesEmbeddingsData | None` and we check that it is not `None` on line 136